src/components/Collection/Series/EditSeriesTabs/NameTab.tsx

```diff
@@ -17,10 +21,66 @@ const NameTab = ({ seriesId }: Props) => {
     refetchOnMountOrArgChange: false,
   });
 
+  const [fetchSeries, seriesResults] = useLazyGetSeriesInfiniteQuery();
+
+  const getAniDbSeries = (): SeriesTitleType[] => {
+    const pages = seriesResults.data?.pages;
+    if (!pages) return [];
+
+    const keys = Object.keys(pages);
+    if (!keys?.length) return [];
+
+    return pages[1];
+  };
+
+  const searchSeries = useMemo(() =>
+    debounce(async () => {
+      await fetchSeries({
+        startsWith: search?.length ? search : undefined,
```

Comment:
Is this condition needed?
Because the `useEffect` has `if (!search) return;`, search will always be non-empty